### ISSUE TITLE
hts_SuperDeduper input filenames containing spaces not supported

### ISSUE THREAD

**Issue by yhoogstrate** [ibest/HTStream#160]
hts_SuperDeduper can not run on FASTQ files containing a space in their filename.

In case anyone else reaches this issue before it has been solved and is not able to change filenames, you can work around this issue by creating a symlink in /tmp with a new filename that has no space.

**Comment by yhoogstrate** [comment]
@dstreett I just tested it and apparantly the problem only takes place with gzipped FASTQ's. Excuse me for the lack of this detail in my first post.

**Comment by yhoogstrate** [comment]
It's really simple. A call to gzopen should be more elegant but I don't know what the implications are for static building and not sure if this fits with the rest of your code (and piping).

**Comment by msettles** [comment]
Not re-opening the issue, but who in their right mind would add a space into a fastq file name, someone has to edit the original file name to add the space. This kind of falls in the should never occur in practice category.

**Comment by yhoogstrate** [comment]
You are absolutely right that changing filenames is really bad practice, but the problem also occurs with spaces in the parental directory names. The latter may be unhandy but not bad by design.
